### DISCUSSION TITLE
Get vibration/rumble on xbox controller when connected to client with bluetotth?

### DISCUSSION THREAD

**Discussion by qstore** [ClassicOldSong/Apollo#54] [Q&A] [answered]
Hey!  
There is any available setup, to get rumble on xbox controller when the host pc (apollo) is too far from TV ; and the client (moonshine) run on a CCWGT and the xbox series x controller connected to chromecast via bluetooth? 

Thanks!

**Comment by ClassicOldSong** [comment, marked as answer]
Rumble with xbox series controllers is available, might be your device's driver doesn't support rumble with bluetooth.

**Reply by qstore** [reply]
Thanks. Under device you mean the Xbox series X controller or the Google chromecast with Google tv? 
As I remember the controller trough Bluetooth make rumble when connected to pc bit not sure.

**Reply by ClassicOldSong** [reply]
The driver on Chromecast.

https://www.reddit.com/r/Chromecast/comments/jvd5n2/no_controller_vibration_with_chromecast_with/

**Comment by qstore** [comment]
Thanks to your work, I got what I've been waiting for for years. Play PC games comfortably on TV. and my PC is humming loudly somewhere far away from me. Can I support you somewhere or buy a virtual coffee?

**Reply by ClassicOldSong** [reply]
The majority work for streaming was done by and original creators of Sunshine and Moonlight, what I've done is improving user experience and functionality based on their previous works. You should consider support them first.

If you still want to support me, you can visit https://su.mk/store and see if there's anything you like.
